resources/measure/measure-cms124.json

```diff
@@ -0,0 +1,217 @@
+  "improvementNotation": "Higher score equals better quality",
+  "guidance": "To ensure the measure is only looking for a cervical cytology test only after a woman turns 21 years of age, the youngest age in the initial population is 23. <br> Patient self-report for procedures as well as diagnostic studies should be recorded in 'Procedure, Performed' template or 'Diagnostic Study, Performed' template in QRDA-1. <br> Include only cytology and HPV \"co-testing\"; in co-testing, both cytology and HPV tests are performed (i.e., the samples are collected and both tests are ordered, regardless of the cytology result) on the same date of service. Do not include reflex testing. In addition, if the medical record indicates the HPV test was performed only after determining the cytology result, this is considered reflex testing and does not meet criteria for the measure.",
+  "group": [
+    {
+      "identifier": {
+        "value": "CMS124-group-1"
+      },
+      "population": [
+        {
+          "identifier": {
+            "value": "initial-population-identifier"
+          },
+          "code": {
+            "coding": [
+              {
+                "code": "initial-population"
+              }
+            ]
+          },
+          "criteria": "CMS124.\"Initial Population\""
+        },
+        {
+          "identifier": {
+            "value": "numerator-identifier"
+          },
+          "code": {
+            "coding": [
+              {
+                "code": "numerator"
+              }
+            ]
+          },
+          "criteria": "CMS124.\"Numerator\""
+        },
+        {
+          "identifier": {
+            "value": "denominator-identifier"
+          },
+          "code": {
+            "coding": [
+              {
+                "code": "denominator"
+              }
+            ]
+          },
+          "criteria": "CMS124.\"Denominator\""
+        },
+        {
+          "identifier": {
+            "value": "denominator-exclusions-identifier"
+          },
+          "code": {
+            "coding": [
+              {
+                "code": "denominator-exclusion"
+              }
+            ]
+          },
+          "criteria": "CMS124.\"Numerator Exclusion\""
+            }
+          ]
+        }
+      ],
+  "supplementalData": [
+    {
+      "identifier": {
+        "value": "supplemental-data-race"
+      },
+      "criteria": "CMS124.\"Supplemental Data - Race\""

```

Comment:
These expression references should be the same as what's in the CQL (e.g. CMS124.\"SDE Race\")